src/Components/Nav/index.js

```diff
@@ -1,9 +1,19 @@
+import { Link, NavLink } from "react-router-dom"
 import React from "react"
 import "./nav.css"
+
 function Nav() {
     return (
         <nav>
```

Comment:
In the wireframe the logo is to the left, links to the right. You should split them in two child divs of nav element and (justify-content: space-between;) on nav